MetadataExtractor/Formats/Png/PngDescriptor.cs

```diff
@@ -82,10 +82,10 @@ public sealed class PngDescriptor(PngDirectory directory)
             if (bytes is null)
                 return null;
 
-            var reader = new SequentialByteArrayReader(bytes);
+            var reader = new BufferReader(bytes, isBigEndian: true);
             try
             {
-                // TODO do we need to normalise these based upon the bit depth?
+                // TODO do we need to normalize these based upon the bit depth?
```

Comment:
Visual studio recomends the en-US form of these words, but let me know if you prefer to keep the en-GB / or Australian spellings on these.

Reply:
I don't mind. This is not my hill to die on :) On my PC though I have en-GB and en-US and don't see warnings. At work I try to use en-US though. Another option is to try and specify both locales in the `.editorconfig` somehow.